src/test/java/org/opensearch/security/sanity/tests/SecurityRestTestCase.java

```diff
@@ -45,6 +45,8 @@ public class SecurityRestTestCase extends OpenSearchRestTestCase {
 
     private static final String CERT_FILE_DIRECTORY = "sanity-tests/";
 
+    static boolean useAdminAsPassword = false;
```

Comment:
Adding static override is a code smell, instead could the test set the property of `password` to be admin?

Reply:
We should delete this

Reply:
refactored.